pandas/core/reshape/reshape.py

```diff
@@ -1089,6 +1089,198 @@ def get_empty_frame(data) -> DataFrame:
         return DataFrame(dummy_mat, index=index, columns=dummy_cols)
 
 
+def from_dummies(
+    data: DataFrame,
+    subset: None | list[str] = None,
```

Comment:
Isn't subset `None | list[Hashable]`?

Reply:
For example, if my columns are `[1, 2, 3, 4]`, then [1, 3, 4] should be ok for subset.

Reply:
Oh you are right, I was too zoomed in on the column names with a prefix to relize that there are other valid column names as well. I will look into this and fix it, thank you :)